connector/src/main/java/it/eng/connector/model/User.java

```diff
@@ -1,16 +1,14 @@
 package it.eng.connector.model;
 
-import java.io.Serial;
 import java.util.Collection;
 import java.util.List;
 
-import jakarta.persistence.Id;
 import org.springframework.data.mongodb.core.mapping.Document;
 import org.springframework.security.core.GrantedAuthority;
 import org.springframework.security.core.authority.SimpleGrantedAuthority;
 import org.springframework.security.core.userdetails.UserDetails;
 
-
+import jakarta.persistence.Id;
```

Comment:
Here I would use: import org.springframework.data.annotation.Id;

Reply:
No problemo, I will change it